src/Http/Response.php

```diff
@@ -831,12 +834,22 @@ protected function _getUTCDate(DateTimeInterface|string|int|null $time = null):
             $result = new DateTime($time ?? 'now');
         }
 
-        /**
-         * @phpstan-ignore-next-line
-         */
+        /** @phpstan-ignore-next-line */
         return $result->setTimezone(new DateTimeZone('UTC'));
     }
 
+    /**
+     * Converts the time zone to GMT and returns a string in RFC7231 format.
+     * This replaced the deprecated and broken ``DATE_RFC7231`` formatting constant.
+s     *
+     * @param DateTimeInterface|string|int|null $time
+     * @return string
+     */
+    protected function _getRfc7231(DateTimeInterface|string|int|null $time = null): string
```

Comment:
We don't need to use underscore prefix for new protected functions, it will have to be changed again for 6.0.